### PR TITLE
flip renderNodesForLane render order

### PR DESCRIPTION
**Pull request by ASBrattsev**
apparently plantuml has changed something, and now you don't need this crutch to render the image correctly. On the contrary, because of this, the rendering is now going from right to left

### CODE REVIEW

**Comment by coulas** [comment]
I applied this patch manually and the rendering went form reversed order to a correct order on my mac with that plantuml:
plantuml --version
PlantUML version 1.2024.8 (Fri Nov 15 18:44:46 CET 2024)
(GPL source distribution)
Java Runtime: OpenJDK Runtime Environment
JVM: OpenJDK 64-Bit Server VM
Default Encoding: UTF-8
Language: fr
Country: FR
 
PLANTUML_LIMIT_SIZE: 4096

Dot version: dot - graphviz version 12.2.1 (20241206.2353)
Installation seems OK. File generation OK

In short, it works on my machine ;-)
